commons/src/test/java/com/ge/predix/acs/utils/HttpServletRequestUtilTest.java

```diff
@@ -84,6 +85,7 @@ public void getSubdomainEmptyDomain() {
         Assert.assertNull(HttpServletRequestUtil.getSubdomain(hostname, domain));
     }
 
+    @SuppressFBWarnings(value = "NP_NULL_PARAM_DEREF_NONVIRTUAL")
```

Comment:
We will have to remove @SuppressFBWarnings even though it means false positive from findbugs report since our black duck scan team is disallowing findBugs annotations usage in our code. I need to talk to Jeff from black duck scan team why they are disallowing findBugs annotation until then we will have to assume that findBugs annotations are dis-allowed.